src/predict.py

```diff
@@ -71,11 +71,16 @@ def prediction(batch_dispatcher, tf_vars, array_cost, pred_array, id_array):
         config = json.load(open(os.path.join(experiment_folder, 'config.json')))
         config['mode'] = 'regular'
 
+        if 'melspectrogram' in config['audio_rep']['type']:
```

Comment:
same as above:
Is there a reason to do the check with `in` or could it also be done with `==`?